substrate/frame/session/src/lib.rs

```diff
@@ -683,12 +697,12 @@ impl<T: Config> Pallet<T> {
 		let session_index = session_index + 1;
 		CurrentIndex::<T>::put(session_index);
 		T::SessionManager::start_session(session_index);
-		log::trace!(target: "runtime::session", "starting_session {:?}", session_index);
+		log!(trace, "starting_session {:?}", session_index);
 
 		// Get next validator set.
 		let maybe_next_validators = T::SessionManager::new_session(session_index + 1);
-		log::trace!(
-			target: "runtime::session",
+		log!(
+			trace,
```

Comment:
here and elsewhere: would be gud to bring the `target: LOG_TARGET` back, otherwise, setting trace level logs will be painful